planning_scene_loader:
  ros__parameters:
    move_group_name: "ur_manipulator"

    table_position: [0.0, 0.0, 0.0] # 0.015 is height of UR5e base plate
    table_scale: [0.9, 1.5, 0.1]
    
    mortar_mesh_file_name: "asone_agate_mortar_80x100x36.stl"
    mortar_top_position: [0.39, 0.366, 0.042]
    mortar_inner_scale: [0.04, 0.04, 0.035] # official information says 0.04 * 0.04 * 0.032 but z is more deep actually
